coreneuron/io/reports/report_handler.cpp

```diff
@@ -155,6 +161,73 @@ VarsToReport ReportHandler::get_compartment_vars_to_report(const NrnThread& nt,
     return vars_to_report;
 }
 
+std::string getSectionTypeStr(SectionType type) {
+    switch (type) {
+        case Axon:
+            return "axon";
+        case Dendrite:
+            return "dend";
+        default:
+            return "apic";
```

Comment:
We can make this future proof:
```suggestion
        case Apical:
            return "apic";
        default:        
            std::cerr << "SectionType not handled in getSectionTypeStr" << std::endl;
            nrn_abort(1);
```

Reply:
👍